ush/config.sh.RRFS_AK_dev1

```diff
@@ -20,6 +20,34 @@ GWD_HRRRsuite_BASEDIR=/mnt/lfs4/BMC/nrtrr/RRFS/fix/fix_lam.20210128
 FIX_GSI=/mnt/lfs4/BMC/nrtrr/RRFS/fix/fix_gsi
 FIX_CRTM=/home/rtrr/FIX_EXEC_MODULE/crtm/CRTM_v2.3.0
 
+if [[ $MACHINE == "hera" ]] ; then
+  ACCOUNT="wrfruc"
+  PARTITION_DEFAULT=""
+  PARTITION_FCST=""
+  QUEUE_ANALYSIS="batch"
+
```

Comment:
No need to set PARTITION_GRAPHICS and PARITION_ANALYSIS  because they has default as "".
But setup.sh try to set the "hereafter" as partition name for PARTITION_DEFAULT, PARTITION_FCST, which caused trouble when I submit the job.